main.go

```diff
@@ -1,13 +1,24 @@
 package main
 
 import (
-	_ "github.com/mdg-iitr/Codephile/conf"
+	"fmt"
+	"os"
+
 	"github.com/astaxie/beego"
-	_ "github.com/mdg-iitr/Codephile/routers"
 	sentryhttp "github.com/getsentry/sentry-go/http"
+	_ "github.com/mdg-iitr/Codephile/conf"
+	_ "github.com/mdg-iitr/Codephile/routers"
 )
 
 func main() {
+	if os.Getenv("PORT") == "8080" {
```

Comment:
Add a new env variable `ENVIRONMENT` then using port

Reply:
possible values be `dev` and  `prod` for now

Reply:
done

Reply:
I cant see this change

Reply:
<img width="286" alt="Screenshot 2021-03-27 at 3 20 08 PM" src="https://user-images.githubusercontent.com/63503364/112717032-f2757f00-8f0f-11eb-907d-1a6a8e82ec14.png">

Reply:
> Add a new env variable `ENVIRONMENT` then using port

Replace `PORT` with  `ENVIRONMENT`

Reply:
> Add a new env variable `ENVIRONMENT` then using port

Replace `PORT` with  `ENVIRONMENT`

Reply:
new variable in env file
<img width="143" alt="Screenshot 2021-03-27 at 3 42 04 PM" src="https://user-images.githubusercontent.com/63503364/112717500-2ef6aa00-8f13-11eb-9e18-7c06b6ca2328.png">
___________________________
<img width="471" alt="Screenshot 2021-03-27 at 3 42 32 PM" src="https://user-images.githubusercontent.com/63503364/112717501-3027d700-8f13-11eb-9e1c-72106ce6741c.png">

Reply:
<img width="525" alt="Screenshot 2021-03-27 at 3 55 14 PM" src="https://user-images.githubusercontent.com/63503364/112717811-e50ec380-8f14-11eb-8c29-9b14b79d5ddf.png">

_________________________________________
new .env variable ENVIRONMENT is required to be added from now on
<img width="143" alt="Screenshot 2021-03-27 at 3 42 04 PM" src="https://user-images.githubusercontent.com/63503364/112717817-ef30c200-8f14-11eb-9f6b-9205dd25d326.png">

hopefully this is correct ?

Reply:
Yup